### DISCUSSION TITLE
로컬 GPU를 사용하는 경우

### DISCUSSION THREAD

**Discussion by FIVEYOUNGWOO** [Management001/How_to_install_CUDA-#2] [Q&A] [answered]
안녕하세요. 해당 방법대로 설치를 진행하고 있는데, 로컬 GPU를 사용하는 경우, 해당 방안대로 설치를 안해도 무방할까요?..

**Comment by Management001** [comment, marked as answer]
네, 가능합니다! 로컬에서 GPU 사용할 수 있도록 하려면 CUDA는 GPU에 맞는 드라이버를 설치하시면 됩니다.

**Reply by FIVEYOUNGWOO** [reply]
알려주셔서 감사합니다 :) 보면서 차근차근 시도해볼게요
